client/header/activity-panel/panels/inbox/style.scss

```diff
@@ -0,0 +1,137 @@
+
+
+.woocommerce-inbox-message {
+	position: relative;
+	color: $gray-text;
+	background: $core-grey-light-200;
+	border-radius: 2px;
```

Comment:
Will this need the vendor properties? or a mixin?

Reply:
Hi @becdetat,
I tried the code in Safari, Firefox, and Chrome and it worked properly, did you find a compatibility problem with a browser?

Reply:
No, looks like it's supported in IE10, I thought it wasn't :-)